examples/ExactlyOnceBasic/Program.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using Microsoft.Extensions.Configuration;
+using Confluent.Kafka;
+
+
+/// <remarks>
+///     Set Sasl credentials by adding to appsettings.json or via the env vars:
+///     KAFKA_EXAMPLE__Common__SaslUsername
+///     KAFKA_EXAMPLE__Common__SaslPassword
+/// </remarks>
+class ProcessConfig
+{
+    public static T GetConfig<T>(string configSection)
```

Comment:
on reflection i think it would be better to instantiate the ProcessConfig class, and not re-read the config every time it's needed, as i'm doing here. i originally figured this was no big deal to be inefficient as it doesn't matter... but since this is serving as an example, we probably should do it properly.